src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import AnimateBlock from '../AnimateBlock';
+
+const qaId = 'animate-block';
+
+describe('AnimateBlock', () => {
+    test('render AnimateBlock by default', async () => {
+        render(<AnimateBlock qa={qaId} />);
+
+        const component = screen.getByTestId(qaId);
+        expect(component).toBeInTheDocument();
+    });
+
+    test('add className', () => {
+        const className = 'my-class';
```

Comment:
Can we use here a variable? We can use it in other tests

Reply:
Let us leave it as it is